Sources/NIO/Embedded.swift

```diff
@@ -630,6 +630,23 @@ public final class EmbeddedChannel: Channel {
         // This will never throw...
         try! register().wait()
     }
+    
+    /// Create a new instance.
+    ///
+    /// During creation it will automatically also register itself on the `EmbeddedEventLoop`.
+    ///
+    /// - parameters:
+    ///     - handlesr: The `ChannelHandler`s to add to the `ChannelPipeline` before register or `nil` if none should be added.
+    ///     - loop: The `EmbeddedEventLoop` to use.
+    public init(handlers: [ChannelHandler], loop: EmbeddedEventLoop = EmbeddedEventLoop()) {
+        self.embeddedEventLoop = loop
```

Comment:
could we have the only `init(handler: ChannelHandler...)` one call this version? There should only be one `init` which does the actual work.